examples/raw_input/RawInput.cpp

```diff
@@ -0,0 +1,82 @@
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Graphics.hpp>
+
+#include <vector>
+
+
+namespace
+{
+std::filesystem::path resourcesDir()
+{
+#ifdef SFML_SYSTEM_IOS
+    return "";
+#else
+    return "resources";
+#endif
+}
+} // namespace
+
+
+////////////////////////////////////////////////////////////
+/// Entry point of application
+///
+/// \return Application exit code
+///
+////////////////////////////////////////////////////////////
+int main()
+{
+    // Create the main window
+    sf::RenderWindow window(sf::VideoMode({800u, 600u}), "SFML Raw Mouse Input", sf::Style::Titlebar | sf::Style::Close);
+    window.setVerticalSyncEnabled(true);
+
+    // Load the application font and pass it to the Effect class
+    const auto font = sf::Font::loadFromFile(resourcesDir() / "tuffy.ttf").value();
+
+    auto vectorToString = [](const auto& vector)
+    { return '(' + std::to_string(vector.x) + ", " + std::to_string(vector.y) + ')'; };
+
+    // Create the mouse position text
+    sf::Text mousePosition(font, "Mouse Position: " + vectorToString(sf::Mouse::getPosition()), 20);
```

Comment:
```suggestion
    sf::Text mousePosition(font, "", 20);
```

Probably not worth stringifying the position here as it will immediately be updated on the next frame, using an empty string here makes it so that we have one single place where the text is updated.